.github/config/codeql-config.yml

```
@@ -5,4 +5,5 @@ paths:
 
 paths-ignore:
   - '**/node_modules'
-  - 'src/Umbraco.Web.UI/wwwroot'
\ No newline at end of file
+  - 'src/Umbraco.Web.UI/wwwroot'
+  - 'src/Umbraco.Cms.StaticAssets'
```

It's probably better to only exclude `wwwroot`, in case we ever put other assets into this project (it already contains cshtml/Razor files):

```suggestion
  - 'src/Umbraco.Cms.StaticAssets/wwwroot'
```